fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -306,7 +314,7 @@ ThreadFuture<Void> DLDatabase::onReady() {
 
 Reference<ITransaction> DLDatabase::createTransaction() {
 	FdbCApi::FDBTransaction* tr;
-	api->databaseCreateTransaction(db, &tr);
+	throwIfError(api->databaseCreateTransaction(db, &tr));
```

Comment:
This gets called from two places: one for MVC legacy version monitoring and the other when calling `updateTransaction`. The latter is called when the transaction is created, calls `onError`, or is reset. I haven't carefully looked at all of these locations, but it looks like at least some of them might not behave the way we want if an exception is thrown to them, so we should update them for this possibility.